core/cmd/remote_client.go

```diff
@@ -463,6 +464,36 @@ func (cli *Client) ConfigDump(c *clipkg.Context) (err error) {
 	return nil
 }
 
+func (cli *Client) ConfigFileValidate(c *clipkg.Context) error {
+	fileName := c.String("file")
+	err := validateConfigFile(fileName)
+	if err != nil {
+		fmt.Printf("Validation failed (%s)\n", err.Error())
```

Comment:
Will this log double, since it is also returned? Can we just `return err` here instead? (and `return nil` below)